src/main/java/Edit_Group_Profile_Use_Case/EditGroupProfileInputBoundary.java

```diff
@@ -0,0 +1,8 @@
+package Edit_Group_Profile_Use_Case;
+import Entities.GroupProfile;
+
+public interface EditGroupProfileInputBoundary {
+    EditGroupProfileResponseModel create(EditGroupProfileRequestModel requestModel);
```

Comment:
Perhaps a more intuitive name for this function would be edit() instead of create()

Reply:
I agree with Sohee. I also did create() as following Paul's example. However, I think that edit() might be too broad of a name. There are many use cases that are "edit"-ing something so it might be a little confusing. Maybe consider editGroup() or something even more specific.

Reply:
Yes, good point helena! Better to have a specific method name